pkg/scheduler/cache/cache.go

```diff
@@ -562,26 +555,9 @@ func (sc *SchedulerCache) Run(stopCh <-chan struct{}) {
 }
 
 // WaitForCacheSync sync the cache with the api server
-func (sc *SchedulerCache) WaitForCacheSync(stopCh <-chan struct{}) bool {
-	return cache.WaitForCacheSync(stopCh,
-		func() []cache.InformerSynced {
-			informerSynced := []cache.InformerSynced{
-				sc.podInformer.Informer().HasSynced,
-				sc.podGroupInformerV1beta1.Informer().HasSynced,
-				sc.nodeInformer.Informer().HasSynced,
-				sc.pvInformer.Informer().HasSynced,
-				sc.pvcInformer.Informer().HasSynced,
-				sc.scInformer.Informer().HasSynced,
-				sc.queueInformerV1beta1.Informer().HasSynced,
-				sc.quotaInformer.Informer().HasSynced,
-				sc.cpuInformer.Informer().HasSynced,
-			}
-			if options.ServerOpts.EnablePriorityClass {
-				informerSynced = append(informerSynced, sc.pcInformer.Informer().HasSynced)
-			}
-			return informerSynced
-		}()...,
-	)
+func (sc *SchedulerCache) WaitForCacheSync(stopCh <-chan struct{}) {
+	sc.informerFactory.WaitForCacheSync(stopCh)
```

Comment:
should we cope with the cacheSync error ?

Reply:
> should we cope with the cacheSync error ?

But so far no functions care about the result of `WaitForCacheSync`

I can add some code to check the result of `informerFactory.WaitForCacheSync` and return bool value,  if that is you want

Reply:
never mind, it's ok to just leave it here